RecoLocalCalo/HGCalRecProducers/plugins/alpaka/HGCalLayerClustersAlgoWrapper.dev.cc

```diff
@@ -0,0 +1,55 @@
+// Check that ALPAKA_HOST_ONLY is not defined during device compilation:
+#ifdef ALPAKA_HOST_ONLY
+#error ALPAKA_HOST_ONLY defined in device compilation
+#endif
+
+#include "RecoLocalCalo/HGCalRecProducers/interface/HGCalTilesConstants.h"
+
+#include "HGCalLayerClustersAlgoWrapper.h"
+#include "ConstantsForClusters.h"
+
+#include "CLUEAlgoAlpaka.h"
+
+namespace ALPAKA_ACCELERATOR_NAMESPACE {
+
+  using namespace cms::alpakatools;
+
```

Comment:
you can also `using namespace hgcal::constants` to avoid using too long names :-)